### PR TITLE
moved some schema bits in from riak.schema

### PR DESCRIPTION
**Pull request by joedevivo**
I found some bits in the riak/rel/files/riak.schema that should be in here.

### CODE REVIEW

File: test/riak_kv_schema_tests.erl [hunk 1]
```diff
@@ -77,7 +81,11 @@ override_non_multi_backend_schema_test() ->
         {["object_format"], v0},
         {["memory_backend", "max_memory"], "8GB"},
         {["memory_backend", "ttl"], "1d"},
-        {["secure_referer_check"], off}
+        {["secure_referer_check"], off},
+        {["warn_object_size"], "10MB"},
+        {["max_object_size"], "100MB"},
+        {["warn_siblings"], 250},
+        {["max_siblings"], 1000}
```
Comment (engelsanchez) [inline test/riak_kv_schema_tests.erl:88]: I have a great idea! If the commas were at the beginning, the merge difference would be cleaner! Let's change all teh codez!

Comment (joedevivo) [inline test/riak_kv_schema_tests.erl:88]: commas get their own lines!

**Comment by engelsanchez** [comment]
+1 I verified building a devrel after removing the bits from riak.schema. Values are set correctly, it's all good.

**Comment by joedevivo** [comment]
Cool. I'll remove those in another PR that includes some other stuff later today

**Pull request merged**